src/include/api.h

```diff
@@ -89,6 +89,15 @@
 #define	TXN_API_END(s, ret)	TXN_API_END_RETRY(s, ret, 1)
 
 /*
+ * In cursor open call we should not change the transaction state.
+ * If method is about to return WT_NOTFOUND (some underlying object was
+ * not found), map it to ENOENT.
+ */
+#define	CURSOR_OPEN_API_END(s, ret)					\
```

Comment:
It would be better if this had a generic name, something like: `API_END_RET_NO_TXN_ERROR`. That matches our other conventions:
`API_END` is a standard prefix
`_RET` is for macros that return directly
`_NO_TXN_ERROR` describes why it's special

It took me quite a while to spot the `0` being passed to `API_END` - I think a better name would make that clearer. There would be a corresponding reworking of the comment to match the generic macro.